src/shell/shell_cmds.cpp

```diff
@@ -835,26 +835,29 @@ void DOS_Shell::CMD_LS(char *args)
 	const int column_sep = 2; // chars separating columns
 	const auto cell_widths = map_name_lengths(results, column_sep);
 	const auto column_widths = calc_column_widths(cell_widths, column_sep + 1);
+	const size_t cols = column_widths.size();
 
 	size_t w_count = 0;
 
 	for (const auto &entry : results) {
 		std::string name = entry.name;
```

Comment:
Similar comments as before.. I'm not sure what `results` are, likewise `entry` is just a generic name for anything in a collection.  Suggest describing what it actually is.